db/migrate/20161101175911_add_organization_listed_at_index.rb

```diff
@@ -0,0 +1,10 @@
+class AddOrganizationListedAtIndex < ActiveRecord::Migration
+  disable_ddl_transaction!
+
+  def change
+    add_index :organizations,
+    :listed_at,
+    where: "(listed_at IS NOT NULL)",
+    algorithm: :concurrently
```

Comment:
Removed `algorithm :concurrently` and moved index definition to table migration.

As far as the rest goes:

```
[4] pry(main)> Organization.where(listed_at: nil).explain
  Organization Load (3.3ms)  SELECT "organizations".* FROM "organizations" WHERE "organizations"."listed_at" IS NULL
=> EXPLAIN for: SELECT "organizations".* FROM "organizations" WHERE "organizations"."listed_at" IS NULL
                           QUERY PLAN
----------------------------------------------------------------
 Seq Scan on organizations  (cost=0.00..14.20 rows=2 width=160)
   Filter: (listed_at IS NULL)
(2 rows)

CREATE INDEX index_organizations_on_listed_at ON organizations USING btree (listed_at) WHERE (listed_at IS NOT NULL);
```

I think that should be an index scan instead of a seq scan, but that's how all the other indices are being built. What would you change?